src/platform/silabs/SiWx917/wifi/wfx_host_events.h

```diff
@@ -240,12 +243,19 @@ void wfx_dhcp_got_ipv4(uint32_t);
 void wfx_ip_changed_notify(int got_ip);
 #endif /* CHIP_DEVICE_CONFIG_ENABLE_IPV4 */
 
+sl_status_t sl_si91x_host_process_data_frame(sl_wifi_interface_t interface, sl_wifi_buffer_t * buffer);
+void * sl_si91x_host_get_buffer_data(sl_wifi_buffer_t * buffer, uint16_t offset, uint16_t * data_length);
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
```

Comment:
same as my other comment. My bad sorry. use `SL_ICD_ENABLED` here

Reply:
changed it to SL_ICD_ENABLED